pkg/cmd/cmd_test.go

```diff
@@ -0,0 +1,20 @@
+// Copyright KubeArchive Authors
+// SPDX-License-Identifier: Apache-2.0
+package cmd
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// loadGoldenFile loads content from a golden file in testdata
+func loadGoldenFile(t *testing.T, filename string) string {
+	t.Helper()
+	path := filepath.Join("testdata", filename)
+	content, err := os.ReadFile(path)
+	require.NoError(t, err, "Failed to read golden file: %s", path)
```

Comment:
Why `require` instead of `assert.NoError`?

Reply:
if I'm not mistaken, `require.NoError` fails faster, with `assert.NoError` the test continues. In this case, this isn't expected to fail and it isn't part of the "test" per se, it's part of the set up of the test so I think it makes more sense to use `require` than `assert`.

Related: https://pkg.go.dev/github.com/stretchr/testify/require#hdr-Assertions